### DISCUSSION TITLE
Will duplicate movies appear in collection if found using multiple lists?

### DISCUSSION THREAD

**Discussion by vitadome** [meisnate12/Plex-Meta-Manager#116] [Q&A] [answered]
If I reference multiple Trakt Lists will multiple instances of the same movie appear in my collection?  For example, if I create a Christmas collection, reference multiple Trakt Lists, and the movie "Elf" appears in both lists, will that movie display twice in my collection?

collections:
  Christmas:
    trakt_list:
      - https://trakt.tv/users/movistapp/lists/christmas-movies  **(ELF is in this list)**
      - https://trakt.tv/users/2borno2b/lists/christmas-movies-extravanganza **(ELF is in this list too)**

**Comment by meisnate12** [comment, marked as answer]
No you're fine having it in multiple lists, the script just adds the collection tag to the movie or show object if it doesn't already have it

**Reply by vitadome** [reply]
I really appreciated your help/guidance!  Thank you!

**Reply by meisnate12** [reply]
No problem let me know if you have anymore questions
